core/src/test/java/io/confluent/rest/ApiHeadersTest.java

```diff
@@ -130,7 +130,7 @@ private static void createKeystoreWithCert(File file, String alias, Map<String,
     final X509Certificate cert = new CertificateBuilder(30, "SHA1withRSA")
         .sanDnsNames("localhost").generate("CN=mymachine.local, O=A client", keypair);
 
-    TestSslUtils.createKeyStore(file.getPath(), new Password(SSL_PASSWORD), alias,
+    TestSslUtils.createKeyStore(file.getPath(), new Password(SSL_PASSWORD), new Password(SSL_PASSWORD), alias,
```

Comment:
This confuses me. It seems like `createKeyStore`'s signature has been stable since 2015, but there have been changes to these tests more recently than that. How could the tests have passed?

Reply:
FYI, although this method has been stable, the previously called method now no longer exists, hence the compile failure. This just bit MDS as well.